### ISSUE TITLE
[BUG] Misaligned max.HP values in Player Sheets on Firefox

### ISSUE THREAD

**Issue by archeresky** [sdenec/tidy5e-sheet#704]
**Module Version:** v0.8.23

**Describe the bug**
On Firefox, the max HP value text is misaligned in Player Sheets, this bug only appears on Firefox so it must be connected to Firefox interpretation of internal css.

**To Reproduce**
n/a

**Expected behavior**
Text should be aligned.

**Screenshots**
![image](https://user-images.githubusercontent.com/73297245/219089218-674f893c-4033-4bb6-85f1-83678ea2c129.png)

**Browser:**
Firefox 110 

**Foundry Version:** 
10.291
**Game System:** 
2.1.4

**Comment by p4535992** [comment]
Seem to work on my firefox version 110, try to clean up the browser cache.

**Comment by archeresky** [comment]
Sorry for late response, cleared cache and start up cache but it's still misaligned. Checked on my laptop and it's misaligned there as well but to a lesser degree, probably due to resolution scaling (Macbook Retina).

<img width="172" alt="image" src="https://user-images.githubusercontent.com/73297245/221391454-dde12f68-daba-4a34-ac9d-8c87538e0c6e.png">

**Comment by archeresky** [comment]
I have found the solution. Deleting `height: 100%;` at line 1220 in tidy5e.css fixes the alignment problem on Firefox. Tested from breakage on other browsers but seems fine.
```css
.tidy5e.sheet.actor .portrait-hp span {
	font-family: var(--t5e-modesto);
	height: 100%; /* needs to be removed or commented out */
	font-weight: 700;
	/* width: 40px; */
}
```

**Comment by p4535992** [comment]
Applied your solution on version 0.8.32

**Comment by archeresky** [comment]
Can confirm as fixed as of 0.8.32
